### PR TITLE
modify lvscare name

### PR DESCRIPTION
**Pull request by Stevent-fei**

### CODE REVIEW

**Comment by kakaZhou719** [comment]
how about make it as a const value , replace it  including registry lvscare and k8s runtime lvscare

File: common/common.go [hunk 1]
```diff
@@ -40,6 +40,7 @@ const (
 	KubeLvsCareStaticPodName = "kube-lvscare"
 	RegLvsCareStaticPodName  = "reg-lvscare"
 	StaticPodDir             = "/etc/kubernetes/manifests"
+	LvsCare                  = "sealerio/lvscare:v1.1.3-beta.8"
```
Comment (kakaZhou719) [inline common/common.go:43]: LvsCareRepoAndTag

File: pkg/registry/local.go [hunk 2]
```diff
@@ -162,7 +162,7 @@ func (c *localConfigurator) configureLvs(registryHosts, clientHosts []net.IP) er
 	}
 
 	//todo should make lvs image name as const value in sealer repo.
-	lvsImageURL := fmt.Sprintf("%s/sealer/lvscare:v1.1.3-beta.8", net.JoinHostPort(c.Domain, strconv.Itoa(c.Port)))
+	lvsImageURL := filepath.Join(net.JoinHostPort(c.Domain, strconv.Itoa(c.Port)), common.LvsCare)
```
Comment (kakaZhou719) [inline pkg/registry/local.go:165]: in cluster runtime, we also need to replace this value.

**Review by Starnop — APPROVED** [review]
LGTM

**Pull request merged**